csharp/IceBox/Greeter/README.md

```diff
@@ -0,0 +1,30 @@
+#  IceBox Greeter
+
+This demo shows how to create an [IceBox][1] service in C#.
+
+You can build the client application and IceBox service with:
+
+``` shell
+dotnet build
+```
+
+To run this demo, you first need to install the `iceboxnet` tool:
+
+```shell
+dotnet tool install iceboxnet --global --prerelease
```

Comment:
We should consider doing a local install:

```
dotnet tool install iceboxnet --prerelease --create-manifest-if-needed
You can invoke the tool from this directory using the following commands: 'dotnet tool run iceboxnet' or 'dotnet iceboxnet'.
Tool 'iceboxnet' (version '3.8.0-nightly.20250305.1') was successfully installed. Entry is added to the manifest file /Users/jose/Documents/3.8/ice-demos/csharp/Ice/Greeter/.config/dotnet-tools.json.
```

If we comment this manifest file, users can do `dotnet tool restore` to install it.

And `dotnet iceboxnet` to run it.

Reply:
I'd rather not commit this manifest file with a reference to a nightly build:
```
cat dotnet-tools.json 
{
  "version": 1,
  "isRoot": true,
  "tools": {
    "iceboxnet": {
      "version": "3.8.0-nightly.20250305.1",
      "commands": [
        "iceboxnet"
      ],
      "rollForward": false
    }
  }
}
```